app/src/main/java/com/example/deejai/RoomSelection/RoomPlayer.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.deejai.RoomSelection
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.TextView
+import android.widget.Toast
+import com.example.deejai.R
+
+class RoomPlayer : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_room_player)
+        var bundle :Bundle ?=intent.extras
+        var message = bundle!!.getString("roomName") // 1
```

Comment:
Remove the comment, it doesn't help to understand the code.

Reply:
done